src/pages/index.tsx

```diff
@@ -1,7 +1,14 @@
 /* eslint-disable import/no-default-export */
 
+import { Button, primary, secondary } from './Button';
+
 const Index = () => {
-  return <div>🤖</div>;
+  return (
+    <div>
+      <Button variant={primary} text="Primary" />
+      <Button variant={secondary} text="Secondary" />
+    </div>
+  );
```

Comment:
We use Storybook for creating basic components. Please do not change the content of this file as it is unnecessary.

Reply:
Done. Thanks for the comment.